spyder_notebook/widgets/client.py

```diff
@@ -123,6 +123,7 @@ def __init__(self, plugin, name):
 
         self.plugin_actions = plugin.get_plugin_actions()
         self.notebookwidget = NotebookWidget(self)
+        self.notebookwidget.selectionChanged.connect(self.hide_header)
```

Comment:
Why does the header is hidden with this signal? I mean, when `selectionChanged` is triggered?